### PR TITLE
CARP-30 | Store transactions to database

### PR DESCRIPTION
**Pull request by davidmisiak**

### CODE REVIEW

**Review by miksax — APPROVED** [review]
LGTM

File: src/queries.rs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        ..Default::default()
+    };
+    let transaction_model = transaction_model.insert(db).await?;
+
+    let mut addresses = HashSet::new();
+    let mut tokens = HashSet::from([ADA_TOKEN.clone()]);
+    for output in transaction.outputs.iter().flatten() {
+        addresses.insert(output.address.clone());
+
+        for token in output.assets.iter().flatten() {
```
Comment (miksax) [inline src/queries.rs:68]: Do you store assets after each transaction?

Comment (davidmisiak) [reply src/queries.rs:68]: Sorry, what do you mean?

Comment (miksax) [reply src/queries.rs:68]: sorry, my fault - I miss one part...

**Pull request merged**